docs/datascience/jupyter-kernel-options.md

```diff
@@ -0,0 +1,43 @@
+MetaSocialImage: images/tutorial/social.png
+---
+
+# Jupyter Kernel Options in VS Code <!-- omit in toc -->
+
+The VS Code notebooks’ kernel picker lists the following categories of connection methods to run the code in your Jupyter notebook:
+- [Jupyter Kernels](#jupyter-kernels)
+- [Python Environments](#python-environments)
+- [Existing Jupyter Server](#existing-jupyter-server)
+
+By default, we will recommend the one you’ve previously used with your notebook, but you can choose to connect to any other Jupyter kernels as shown below.
+
+## Jupyter Kernels
+
+The **Jupyter Kernels** category lists all Jupyter kernels that VS Code detects in the context of the compute system it’s operating in (your desktop, Codespaces, remote server, etc.). Each Jupyter kernels has Jupyter [kernel specification](https://jupyter-client.readthedocs.io/en/stable/kernels.html#kernel-specs), or Jupyter kernelspec, which contains a JSON file (i.e., `kernel.json`) with details about the kernel—name, description, CLI information required to launch a process as a kernel.
+
+## Python Environments
+
+The **Python Environments** category lists the Python environments that VS Code detects from the compute system it’s operating in (your desktop, Codespaces, remote server, etc.). It shows all Python environments grouped by type (e.g., conda, venv)—whether the [IPyKernel](https://ipython.readthedocs.io/en/stable/install/kernel_install.html) is installed or not.
+
+   > **Note**: You **__do not__** need to install [jupyter](https://pypi.org/project/jupyter/) into the Python environment you want to use. Only IPython and IPyKernel are required to launch a Python process as a kernel and execute code against your notebook, e.g., `pip install ipython ipykernel`. Visit our [wiki](https://github.com/microsoft/vscode-jupyter/wiki/Kernels-(Architecture)) to learn more.
+
+## Existing Jupyter Server
+
+The **Existing Jupyter Server** category lists remote Jupyter servers previously connected. You can also use this option to connect to an existing Jupyter server running remotely or locally. Grab the URL for your Jupyter server, e.g., `http://<ip-address>:<port>/?token=<token>` and paste it in the `Enter the URL of the running Jupyter server` option to connect to the remote server and execute code against your notebook using that server.
+
+![Enter server URL](images/jupyter-kernel-options/select-enter-server-url.png)
+
+   > **Note**: When you’re starting the Jupyter server, ensure to: (1) allow all origins (e.g., `--NotebookApp.allow_origin = '*'`) to allow your servers to be accessed externally, and (2) set the notebook to listen to listen on all IPs (e.g., `--NotebookApp.ip = '0.0.0.0'`).
+
+Once connected, all active Jupyter sessions will appear on this list. You can create new session from the server’s kernelspec by selecting the `Notebook: Select Notebook Kernel` command -> `Select Another Kernel…` -> `Local Kernel Specs…` -> your server.
```

Comment:
I dont think this is correct, the option should be
 `Notebook: Select Notebook Kernel` command -> `Select Another Kernel…` -> `Existing Jupyter Server…` -> your server.

@minsa110 you might want to try this at your end to ensure we're on the same page with this workflow

Reply:
Confirmed and fixed :)